samples/unixdomainsockets/src/main/java/okhttp3/unixdomainsockets/UnixDomainSocketFactory.java

```diff
@@ -38,7 +38,12 @@ public UnixDomainSocketFactory(File path) {
 
   @Override public Socket createSocket(String host, int port) throws IOException {
     Socket result = createSocket();
-    result.connect(new InetSocketAddress(host, port));
+
+    try {
+      result.connect(new InetSocketAddress(host, port));
+    } catch (IOException e) {
+      result.close();
```

Comment:
This is swallowing the exception and continuing to use it.

Reply:
> This is swallowing the exception and continuing to use it.

You are right. I corrected it.